README.md

```diff
@@ -1,2 +1,4 @@
 # emerald
  A simple Discord bot built with Discord.js and Commando.
+
+        [![Run on Repl.it](https://repl.it/badge/github/EMPERRORPK007/emerald)](https://repl.it/github/EMPERRORPK007/emerald)
```

Comment:
Wouldn't this need to point to the main repo to stay updated according to master?